docs/source/installation.rst

```diff
@@ -179,7 +179,6 @@ In the ``anaconda prompt`` terminal:
 
 6. Build an editable version of sktime :code:`pip install -e .[dev]`
 In order to install all soft dependencies, use :code:`pip install -e .[dev]`
```

Comment:
I've a small objection to the use of word `all`. Just skipping it, or changing it explaining that it's for code quality and testing dependencies, will be preferable to me.

Reply:
looking at this, the command doesn't actually install the soft dependencies. Is this a copy-pastes mistake? Should this not include `all_extras`?